File: app/src/main/java/com/udacity/maluleque/meutako/utils/NumberUtils.java
package com.udacity.maluleque.meutako.utils;

import java.text.NumberFormat;
import java.util.Locale;

public class NumberUtils {
    public static String getFormattedAmount(double amount) {
        return NumberFormat.getCurrencyInstance(Locale.getDefault()).format(amount);
    }
}

File: app/src/main/java/com/udacity/maluleque/meutako/utils/NumberUtils.kt
package com.udacity.maluleque.meutako.utils

import java.text.NumberFormat
import java.util.*

object NumberUtils {
    @JvmStatic
    fun getFormattedAmount(amount: Double): String {
        return NumberFormat.getCurrencyInstance(Locale.getDefault()).format(amount)
    }
}